playbook/app/pb_kits/playbook/utilities/globalProps.ts

```diff
@@ -154,3 +156,24 @@ export const deprecatedProps = (kit: string, props: string[] = []): void => {
     })
   }
 }
+
+export const domSafeProps = (props: {[key: string]: string}): {[key: string]: string} => {
+  const notSafeProps = [
+    'marginRight',
+    'marginLeft',
+    'marginTop',
+    'marginBottom',
+    'marginX',
+    'marginY',
+    'margin',
+    'paddingRight',
+    'paddingLeft',
+    'paddingTop',
+    'paddingBottom',
+    'paddingX',
+    'paddingY',
+    'padding',
+    'dark',
+  ]
+  return omit(props, notSafeProps)
```

Comment:
Concerns with using `lodash`? This was the most readable that I could get. To do it with vanila js it's not be that readable.